lib/shared/addon/components/cru-node-pools/component.js

```diff
@@ -179,6 +180,10 @@ export default Component.extend({
     });
   }),
 
+  nodeTemplateIdChanged: observer('nodePools.@each.{nodeTemplateId}', function() {
```

Comment:
If there is a single template it looks like it's automatically selected... which somehow results in this not firing